lesson03/assignment/css/_components.content.scss

```diff
@@ -1,5 +1,8 @@
+/*Body Element Styling*/
 html {
     box-sizing: border-box;
+    font-size: 62.5%;
+    /*1rem = 10px, allowing for easy font size calculation*/
```

Comment:
What an interesting idea... Does this work as expected when you test by changing your browser's font size, or zoom in or out on the page?